packages/artifact/RELEASES.md

```diff
@@ -94,3 +94,8 @@
 ### 1.1.1
 
 - Fixed a bug in Node16 where if an HTTP download finished too quickly (<1ms, e.g. when it's mocked) we attempt to delete a temp file that has not been created yet [#1278](https://github.com/actions/toolkit/pull/1278/commits/b9de68a590daf37c6747e38d3cb4f1dd2cfb791c)
+
+### 2.0.0
+
+Major release. Supports new Artifact backend for improved speed, reliability and behavior.
+Numerous API changes, some breaking.
```

Comment:
Maybe link to the `Breaking Changes` section of the README?

Reply:
That's a good idea. I'll do that, and also we'll link to whatever blog post we create 👍

Reply:
https://github.com/actions/toolkit/pull/1594/commits/19d4d9d3b25b410ffa72b1e6e1bf4ce4122eedcb